test/express-error.js

```diff
@@ -12,7 +12,7 @@ var root = __dirname + '/public',
 
 mkdirp.sync(root + '/emptyDir');
 
-var cases = require('./common-cases-error');
+var cases = require('./secret/common-cases-error');
```

Comment:
Or to actually answer your question: Yes, tap ignores sub folders, if your glob pattern does.